### PR TITLE
Update community.yml

### LINKED ISSUE #43: Update Community.yml
I will update the issue-message, Pr-Message, with new ones and add a footer message in Community.yml

### PR DESCRIPTION
**Pull request by muthuannamalai12**
I have updated the issue-message, Pr-Message with new ones, and added a footer in Community.yml

### CODE REVIEW

**Review by BOLT04 — COMMENTED** [review]
Thank you for your contribution @muthuannamalai12 👍, but what do you think about making this PR on the `template` repo? The [file on that repo](https://github.com/EddieHubCommunity/template/blob/main/.github/workflows/community.yml) should be used in all others I believe.

It's easier to keep all workflows consistent if we start using them from the `template` repo.
Let me know what you think 😃.

File: .github/workflows/community.yml [hunk 1]
```diff
@@ -20,8 +20,9 @@ jobs:
       - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: '<h1>It''s great having you contribute to this project:tada:</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face:'
-          pr-message: '<h1>It''s great having you contribute to this project:tada:</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face:'
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```
Comment (Vyvy-vi) [inline .github/workflows/community.yml]: Could we use the verbiage- Discord Server and but the hyperlink on Discord Server for consistency?
```suggestion
          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord Server</a> and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
```

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

**Review by Vyvy-vi — APPROVED** [review]
LGTM-!

**Review by galahad42 — APPROVED** [review]

**Comment by eddiejaoude** [comment]
Thanks everyone 👍

**Review by eddiejaoude — APPROVED** [review]
Great work everyone 👍

**Pull request merged**